test/ci-cd/terraform/cloudbuild/cloudbuild_trigger.tf

```diff
@@ -389,6 +389,73 @@ resource "google_cloudbuild_trigger" "platforms_gke_base_uc_federated_learning_s
   }
 }
 
+resource "google_cloudbuild_trigger" "platforms_gke_base_uc_federated_learning_cross_device_scripts" {
+  filename = "test/ci-cd/cloudbuild/uc-federated-learning-cross-device-terraform.yaml"
+  ignored_files = [
+    "platforms/gke/base/core/README.md",
+    "platforms/gke/base/use-cases/federated-learning/README.md",
+    "platforms/gke/base/use-cases/federated-learning/cross-device/README.md"
+  ]
+  included_files = [
+    # Include the whole core platform because we want to ensure that
+    # changes to the base platform don't break this use case
+    "platforms/gke/base/core/**",
+    "platforms/gke/base/use-cases/federated-learning/**",
```

Comment:
Ideally we should only include the terraservices directories that are used in the use cases. See `platforms_gke_base_uc_inference_ref_arch_scripts` below. Otherwise we are needlessly triggering CI which can take some time to run.

Reply:
Yes, just replicate the changes to `platforms_gke_base_uc_federated_learning_cross_device_scripts` as well. I didn't review all of the Terraservices you use, but I assume you included the necessary ones.